install/dc-detect-version.sh

```diff
@@ -9,22 +9,20 @@ fi
 echo "${_group}Initializing Docker Compose ..."
 
 # To support users that are symlinking to docker-compose
-dc_base="$(docker compose version &>/dev/null && echo 'docker compose' || echo 'docker-compose')"
+dc_base="$(docker compose version --short &>/dev/null && echo 'docker compose' || echo '')"
 dc_base_standalone="$(docker-compose version &>/dev/null && echo 'docker-compose' || echo '')"
 
-COMPOSE_VERSION=$($dc_base version --short || echo '')
-STANDALONE_COMPOSE_VERSION=$($dc_base_standalone version --short &>/dev/null || echo '')
+COMPOSE_VERSION=$([ -n "$dc_base" ] && $dc_base version --short || echo '')
```

Comment:
Personally, I think this is more clear for this line and the one below it, but I will leave that up to you:

```
if [ -n "$dc_base" ]; then
  COMPOSE_VERSION=$($dc_base version --short || echo '')
fi
```

Reply:
I'm ok with it as it is